modules/wallet/seed.go

```diff
@@ -42,6 +44,27 @@ func generateSpendableKey(seed modules.Seed, index uint64) spendableKey {
 	}
 }
 
+// generateKeys generates n keys from seed, starting from index start.
+func generateKeys(seed modules.Seed, start, n uint64) []spendableKey {
+	// generate in parallel, one goroutine per core.
+	keys := make([]spendableKey, n)
+	var wg sync.WaitGroup
+	wg.Add(runtime.NumCPU())
+	for cpu := 0; cpu < runtime.NumCPU(); cpu++ {
+		go func(start uint64) {
```

Comment:
Perhaps rename this local parameter `start` to `offset` to avoid confusion with the outer function's `start` parameter.